bears/csv/CSVLintBear.py

```diff
@@ -17,7 +17,80 @@ class CSVLintBear:
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Syntax'}
+    ASCIINEMA_URL = 'https://asciinema.org/a/8fmp2pny34kpqw7t1eoy7phhc'
+
+    regex = re.compile(r'\n\d+\.\s(?P<origin>(?P<severity>\w+))\.\s'
+                       r'(Row:\s(?P<line>[0-9]+)\.\s)?(?P<message>.*)?')
+
+    severity_map = {
+        'wrong_content_type': RESULT_SEVERITY.MAJOR,
+        'ragged_rows': RESULT_SEVERITY.MAJOR,
+        'blank_rows': RESULT_SEVERITY.MAJOR,
+        'invalid_encoding': RESULT_SEVERITY.MAJOR,
+        'not_found': RESULT_SEVERITY.MAJOR,
+        'stray_quote': RESULT_SEVERITY.MAJOR,
+        'unclosed_quote': RESULT_SEVERITY.MAJOR,
+        'whitespace': RESULT_SEVERITY.MAJOR,
+        'line_breaks': RESULT_SEVERITY.MAJOR,
+        'no_encoding': RESULT_SEVERITY.NORMAL,
+        'encoding': RESULT_SEVERITY.NORMAL,
+        'no_content_type': RESULT_SEVERITY.NORMAL,
+        'excel': RESULT_SEVERITY.NORMAL,
+        'check_options': RESULT_SEVERITY.NORMAL,
+        'inconsistent_values': RESULT_SEVERITY.NORMAL,
+        'empty_column_name': RESULT_SEVERITY.NORMAL,
+        'duplicate_column_name': RESULT_SEVERITY.NORMAL,
+        'title_row': RESULT_SEVERITY.NORMAL,
+        'nonrfc_line_breaks': RESULT_SEVERITY.INFO,
+        'assumed_header': RESULT_SEVERITY.INFO}
+
+    message_dict = {
+        'wrong_content_type': 'content type is not text/csv',
```

Comment:
let's make those sentences? Like starting with an upper case letter and ending with a period.